quote-book/test-suite/src/lib.rs

```diff
@@ -30,13 +31,15 @@ pub fn create_sci_builder(
     base_amount: u64,
     counter_amount: u64,
     rng: &mut (impl RngCore + CryptoRng),
+    ledger_db: Option<LedgerDB>,
```

Comment:
Nit: I suggest switching to `Option<&LedgerDB>` in all these create_ methods, and do the cloning inside. It will save typing a whole bunch of `.clone()` calls everywhere.